base_comment_template/models/base_comment_template.py

```diff
@@ -110,12 +86,14 @@ class BaseCommentTemplate(models.Model):
     )
     partner_ids = fields.Many2many(
         comodel_name="res.partner",
+        relation="res_partner_base_comment_template_rel",
+        column1="pid",
```

Comment:
Why using this ugly column names. Use `partner_id`. Anyway, you must reuse the same existing relation and columns if you don't want to add migration scripts for moving data previous to this update. Anyways, you must take care also about the migration from v12 to v13.

Reply:
Ok thanks, i have changed it according to previous relation.